content/en/docs/ambient/getting-started/enforce-auth-policies/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Enforce authorization policies
+description: Enforce Layer 4 and Layer 7 authorization policies in an ambient mesh.
+weight: 4
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+After you have added your application to the ambient mesh, you can secure application access using Layer 4 authorization policies.
```

Comment:
We haven't used the term 'ztunnel' once throughout, I wonder if we should introduce the concept to explain why people now need to think about waypoiints